cmd/checkmarxExecuteScan.go

```diff
@@ -651,22 +676,26 @@ func zipFolder(source string, zipFile io.Writer, patterns []string) error {
 	return err
 }
 
+// filterFileGlob checks if file path matches one of the patterns.
+// If it matches a negative pattern ('!') then false is returned.
+//
+// If it is a directory, false is returned.
+// If no patterns are provided, false is returned.
 func filterFileGlob(patterns []string, path string, info os.FileInfo) bool {
+	if len(patterns) == 0 || info.IsDir() {
+		return false
+	}
+
 	for _, pattern := range patterns {
 		negative := false
 		if strings.HasPrefix(pattern, "!") {
 			pattern = strings.TrimLeft(pattern, "!")
 			negative = true
 		}
-		match, _ := doublestar.Match(pattern, path)
-		if !info.IsDir() {
-			if match && negative {
-				return true
-			} else if match && !negative {
-				return false
-			}
-		} else {
-			return false
+		match, _ := doublestar.PathMatch(pattern, path)
```

Comment:
Was already like that before, but woulnd't it be better to handle the error?

Reply:
done